docs: emphasize some points on what makes a good audit (#10376)
GoogleChrome/lighthouse: docs/new-audits.md

- Contribute significantly towards making the mobile web experience better for end users. 
- Not have a significant impact on our runtime performance or bundle size. 

> - Something that is new, and not something that is already measured by existing audits. 

Author: This is another way to say "we have veto control on what lands in core", which goes without saying. low-value to have here.

- Measurable (especially for performance audits) or have clear pass/fail states.

> - Actionable - when failing, specific advice should be given. If the failure can be tied to a specific resource (a DOM element, script, line of code), use the [appropriate detail type](https://github.com/GoogleChrome/lighthouse/blob/master/types/audit-details.d.ts). If multiple failures can occur for a page, return a table.

Reviewer: ```suggestion
- Actionable - when failing, specific advice should be given. If the failure can be tied to a specific resource (a DOM element, script, line of code), use the appropriate detail type (see below). If multiple failures can occur for a page, return a table.
```

- Not use 3rd party APIs for completing the audit check. 

> ## Actionable

Reviewer: ```suggestion
## Actionability
```

> ## Actionable

Reviewer: actuallly... lets put this actionability section in the plugins.md.  that has a bigger audience than this doc.  we can of course add a link from new-audits to the section.

Author: I already added a link from the plugin doc -> new audit doc. I don't think this file should exist if we put important new-audit advice elsewhere. weird bifurcation imo

Reviewer: we have two docs for two "conceivable" audiences:

* audience who wants to build a plugin full of their own audits
* audience who wants to propose a new audit to core (but doesnt want to implement as a plugin)

The actionability stuff is relevant to both of them.

I'm just saying the 1st audience is bigger than the second.  Really I think this actionability stuff is a Must-read for both audiences but i'd rather the plugin folks see it.  The risk is greater for them; they could end up making a useless audit. Whereas the proposers just need us to provide feedback to improve their proposal.

The new-audit doc still stands on its own afterwards.

i'm happy to do it if you'd rather not spend the time. (though i would like to convince you its better this way. :)

> 1. When failing, specific advice should be given. If an audit can fail in multiple ways, each way should have a specific error message.

Reviewer: i dont think its an "error message" because that indicates a runtime problem with the audit.

i dont know what the right words are but.. "users should understand the specifics of why this case is failing"

1. If the failure can be applied to a specific resource, use the appropriate detail type (see subsection).
1. If multiple failures can occur on a single page, return each (use a table - don't just return a binary score).

> ## Detail Types

Reviewer: ```suggestion
### Detail Types
```
